src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -287,9 +287,9 @@ def load_arguments(self, _):
             c.argument('scope', help="The scope the managed identity has access to")
             c.argument('role', help="Role name or id the managed identity will be assigned")
         with self.argument_context(scope + ' identity assign') as c:
-            c.argument('assign_identities', options_list=['--identities'], nargs='*', help="Space-separated identities to assign. Use '{0}' to refer to the system assigned identity. Default: '{0}'".format(MSI_LOCAL_ID))
+            c.argument('assign_identities', options_list=['--identities'], nargs='*', help="Space-separated identities to assign. For User Assigned Identities, pass ids as full qualified resource ids like '/subscriptions/subscriptionId/resourceGroups/resourceGroupName/providers/Microsoft.ManagedIdentity/userAssignedIdentities/identityName'. Use '{0}' to refer to the system assigned identity. For User Assigned Identities, pass ids as full qualified resource ids like '/subscriptions/subscriptionId/resourceGroups/resourceGroupName/providers/Microsoft.ManagedIdentity/userAssignedIdentities/identityName'. Default: '{0}'".format(MSI_LOCAL_ID))
```

Comment:
Personally, I think adding help examples is better than adding too many descriptions directly in the help message

Reply:
@zhoxing-ms Ok. got it.  Do you want me to update the examples instead.

Reply:
Yes, exactly~